### DISCUSSION TITLE
Can not make WOL (Wake On Lan) work in demo-distro based on jetson 36.3

### DISCUSSION THREAD

**Discussion by AndersGnistrup** [OE4T/meta-tegra#1690] [Q&A] [answered]
I have a problem making the Wake On Lan work. It also seems that any other wake up source has stopped working.
How to I test:
ethtool -s eth0 wol g
echo mem > /sys/power/state

And then on host connected
 etherwake -i   \<NIC\> \<dut MAC\>

This worked on scartgath 35.5 branch but has stopped working on latest scartgath branch.

I have done some digging and my best gees is that some kernel module is missing or some setup has changed. But I just can't figure out what.
There are two lines of interest during suspend.
[  218.369266] tegra194-pcie 140a0000.pcie: Link didn't transition to L2 state
[  218.489325] tegra194-pcie 140a0000.pcie: Link didn't go to detect state

So does anyone have a hint for this issue.
I also tried using the rtc but the result seems to be the same.
https://forums.developer.nvidia.com/t/sleep-state/68201/17

**Comment by madisongh** [comment, marked as answer]
Based on [this dev forum post](https://forums.developer.nvidia.com/t/wake-on-lan-support-for-jetson-orin-nx-16gb-on-l4t-36-3-with-jetpack-6/296145), this sounds like a bug/regression in L4T R36.3.0.

**Reply by AndersGnistrup** [reply]
Agree.
I just downloaded the official NVIDA 36.3 jetson SD image and tested Wake On Lan.
It does not work
Nither does the keyboard/mouse work. There is two differenses.
The lines 
[ 218.369266] tegra194-pcie 140a0000.pcie: Link didn't transition to L2 state
[ 218.489325] tegra194-pcie 140a0000.pcie: Link didn't go to detect state
These are not present when using the NVIDIA ubuntu image.

When using the power button the tegra distro wakes up but shortly after enters sleep again. When using the NVIDIA ubuntu image the system wakes up but does not enter a sleep mode.
